src/main/groovy/life/qbic/datamodel/dtos/projectmanagement/ProjectSpace.groovy

```diff
@@ -1,17 +1,19 @@
 package life.qbic.datamodel.dtos.projectmanagement
 
 /**
- * Represents a QBiC project space.
+ * <h1>Represents a QBiC project space</h1>
  *
- * A space is a logical grouping of projects that have the same context.
+ * <p>A space is a logical grouping of projects that have the same context.</p>
  *
- * A project space has a name that follows this convention:
- *
- *  1. No white space
- *  2. Capital letters only
- *  3. Inner white space is replaced by `_`
- *  4. Trailing and leading white space are trimmed
+ * <p>A project space has a name that follows this convention:</p>
  *
+ * <ul>
+ *  <li> No white space </li>
+ *  <li> Capital letters only </li>
+ *  <li> Inner white space is replaced by `_` </li>
+ *  <li> Trailing and leading white space are trimmed </li>
+ *  <li> Dashes are replaced by `_` </li>
+ * </ul>
```

Comment:
ul is the unordered list and results in bullet points, if you want to have an ordered list just as before you need to use ol
```suggestion
 * <ol>
 *  <li> No white space </li>
 *  <li> Capital letters only </li>
 *  <li> Inner white space is replaced by `_` </li>
 *  <li> Trailing and leading white space are trimmed </li>
 *  <li> Dashes are replaced by `_` </li>
 * </ol>
```

Reply:
thank you @jenniferboedker . Yes i figured out that the order is not necessary, so i went for ul.